.env.test

```diff
@@ -66,3 +66,5 @@ FROM_EMAIL=test@example.com
 GOVUK_NOTIFY_PLAIN_EMAIL_TEMPLATE_ID=example-id
 EMAIL_FILE_PATH='/app/mail'
 USE_STREAMING=False
+COMPRESSION_ENABLED=False
+AWS_REGION=eu-west-
```

Comment:
missing `2` on the end of region var

Reply:
[7679a9e](https://github.com/i-dot-ai/redbox-copilot/pull/341/commits/7679a9ea62cd0e0bfba4f5599c2fd5fc4e0107d2)